### ISSUE TITLE
Members only resources

### ISSUE THREAD

**Issue by angievanhegan** [MantaRayMedia/bgs#101]
# Done criteria
- [ ] Editor can label resources as members only
- [ ] Member only resources can only be viewed when logged in
- [ ] Member resources had a label when they appear in the resources search results
- [ ] Member resources had a label when they appear on the resource page as related resources
- [ ] Member only resources appear in search results and as related resources for all users, but non logged in users only see:
  - [ ] Card content
  - [ ] Title
  - [ ] Resource type
  - [ ] Published date
  - [ ] Last update date
  - [ ] Summary field (this is longer than the card content summary)
  - [ ] Author
  - [ ] Tags

**Comment by angievanhegan** [comment]
@StephMSlack This is ready for you to review.

**Comment by StephMSlack** [comment]
@angievanhegan - is this in drupal? Not exactly sure where I should be looking at things to sign it off.

**Comment by angievanhegan** [comment]
@StephMSlack Yes, it's all functionality in Drupal. Shall we talk through it when you're back in the UK? It all seems to work fine to me.

**Comment by StephMSlack** [comment]
@angievanhegan @ican-icant 
- should the members only icon should show information text (like the other icons here)
- The text "Members only content. Please log in." is not aligned with the rest of the content on the page. The text does not stand out & feels too small (especially in comparison with the larger intro text above)
- Can this members only message be changed by BGS? The current text needs to be changed

- Feedback on this resource should not appear if the user cannot see the content (It looks odd)
![screen shot 2018-02-05 at 15 37 15](https://user-images.githubusercontent.com/32127102/35813087-9923bd7a-0a8a-11e8-89b3-3833bcc0bb6b.png)

**Comment by StephMSlack** [comment]
@ican-icant - do you have some alternate logins I can use to test this point - "Member only resources are only visible by users with the member role when logged in (not other non member roles)

**Comment by StephMSlack** [comment]
@ican-icant when logged out and viewing a member only resource, we should not be able to see/download the resource in the RHS - this is also member only content.
![screen shot 2018-02-14 at 13 29 53](https://user-images.githubusercontent.com/32127102/36206678-30386da8-118b-11e8-866d-27d96549c364.png)

**Comment by angievanhegan** [comment]
@StephMSlack 

- I don't think the members only tag needs any explanation
- What would you like the button text to prompt people to login to view the resource to say? We'll need to edit it. I've suggested 'Log in to view'

**Comment by angievanhegan** [comment]
@dilwarhussain please can you remove the full stop on the button? Thanks

@StephMSlack this is ready for you to sign off.

**Comment by StephMSlack** [comment]
@angievanhegan just going through the final elements of this. Could you confirm which field is used for the download text? At the moment it seems to be using the document name but there is a field in the CMS which says it 'The description may be used as the label of the link to the file.' Some of the document names aren't great, so if we could use the text in the label field that would be great.

**Comment by angievanhegan** [comment]
@StephMSlack Yes, it does go where you think. I've raised another issue to sort out the alignment on this Drupal page. 

![screen shot 2018-02-27 at 13 13 47](https://user-images.githubusercontent.com/3796718/36746355-24ad7bd8-1bc0-11e8-8b8d-869eb0d2870f.png)

**Comment by StephMSlack** [comment]
@angievanhegan - its not just the alignment in drupal - could that field be the one we use for the link instead of the name of the downloadable file?

**Comment by StephMSlack** [comment]
HI @angievanhegan for some reason that isnt working on one of my resources. 2 screen shots to show what I mean
![screen shot 2018-02-28 at 13 54 36](https://user-images.githubusercontent.com/32127102/36791249-2d6dcbae-1c8f-11e8-828b-9c1bdcef5473.png)
![screen shot 2018-02-28 at 13 54 54](https://user-images.githubusercontent.com/32127102/36791253-310055c0-1c8f-11e8-9e88-5b3705bd4dd0.png)

**Comment by angievanhegan** [comment]
@StephMSlack I think this is a bug. I'm going to log it as a new issue.

**Comment by StephMSlack** [comment]
@angievanhegan I'm having problems with
